# IN-BIOS5000 / IN-BIOS9000 Autumn 2024

## Sequencing technologies, data analysis and applications

The information below is preliminary and will be updated.


## Introduction

Here you will find information about the autumn 2024 edition of the credited courses [**IN-BIOS5000**](http://www.uio.no/studier/emner/matnat/ifi/IN-BIOS5000) (MSc level) and [**IN-BIOS9000**](http://www.uio.no/studier/emner/matnat/ifi/IN-BIOS9000) (PhD level), on Sequencing technologies, data analysis and applications.
The course is offered by the Department of Informatics and the Department of Biosciences at the University of Oslo (UiO) in collaboration with contributors from Oslo University Hospital, the Norwegian Veterinary Institute, and the Department of Pharmacy and the Centre for Molecular Medicine Norway (NCMM) at the University of Oslo.
It is partially funded by the [**NORBIS, the national research school in bioinformatics, biostatistics and systems biology**](https://norbis.w.uib.no/in-bios9000-genome-sequencing-technologies-assembly-variant-calling-and-statistical-genomics-2024/).
This course consists of two weeks of lectures and practicals, a home project with an oral exam presentation, and a digital written exam.


## Audience

The course is open for master and PhD students at UiO, as well as to PhD students registered through the NORBIS national research school. A maximum of 30 students in total will be admitted to the course.


## Registering for the course

All students at UiO should register as usual through [Studentweb](https://studentweb.uio.no/).

To earn credits for the course, PhD students from other institutions than UiO must obtain the status as a [visiting PhD candidate](https://www.mn.uio.no/english/research/phd/educational-component/visiting-phd.html) at the UiO Faculty of Mathematics and Natural Sciences. Applications may be submitted starting 14 June 2024 and must be submitted by 1 August 2024.

PhD students from outside UiO [registering through NORBIS](https://norbis.w.uib.no/in-bios9000-genome-sequencing-technologies-assembly-variant-calling-and-statistical-genomics-2024/) are given an extended deadline to apply for visiting PhD candidate status. All of the following information and documents must be sent to the NORBIS course coordinator by Monday 16 September 2024 at the very latest:

- Full name (surname, first name)
- Full Norwegian birth and personal number (possibly D-number)
- Citizenship
- Full address (in home country)
- Mobile telephone number
- Email address
- Documentation that you have been admitted to a PhD programme
- Documentation that IN-BIOS9000 is included in the approved PhD plan. It should be sufficient for the administration or a supervisor at your home institution (the university at which you have been admitted) to confirm this with a letter/email.


## Course content

The course provides knowledge of sequencing technologies and hands-on experience with the analysis of data from several current sequencing platforms and for various biological and medical applications. Tools, algorithms, and statistical methods for data analysis will be included, as well as data quality assessment and analysis reproducibility. The applications that will be covered may vary from year to year but will typically include de novo whole-genome assembly and variant calling. Other applications like RNA-Seq, ChIP-Seq, or metagenomics could also be covered.


## Learning outcome

After completing this course, you will:

* understand the differences, benefits, and drawbacks of the current sequencing technologies, and be able to decide which platform to use in what way for the different applications
* be able to evaluate sequencing data quality and quantity
* know the tools, algorithms, and statistical methods involved in the data analysis
* be able to perform bioinformatics analysis with data tailored towards selected applications
* be able to report on the data analysis in such a way that the methods used and the steps taken are transparent, thus enhancing reproducibility
* be aware of, and know to deal with, the ethical and data-sensitivity issues surrounding sequencing data derived from human subjects
* be able to critically evaluate, validate and judge the results of bioinformatics analysis of sequencing experiments in terms of underlying assumptions, reliability, sensitivity and specificity, and evaluate their value for answering biological questions


## Location

The course will be held at the UiO campus, in seminar room 1456 *Shell*, on floor 1 in the northern part of
[Ole-Johan Dahl's building](https://www.uio.no/english/about/getting-around/areas/gaustad/ga06/), Gaustadalléen 23B, Oslo.


## Schedule (preliminary)

Lectures and practical sessions will be held Monday to Friday, October 28th to November 8th, 2024 (weeks 44 and 45). Course days are usually from 09.15 to 16.00 or 17.00. There will be a lunch break each day from 12.00 to 13.15.

In the table below, teachers are indicated with their initials and listed in the following section.

| Time          | Session (Teacher) |
| ------------- | ----------------- |
| **Week 44**
| **Mon 28 Oct**
| 09.15 - 09.30 | [Welcome and practical information (TR)](Lectures/D01T0915-Welcome_and_practical_information-TR-2024.pdf)
| 09.30 - 12.00 | Fundamentals of Molecular Biology (RS):<br/>[A guided tour of the (human) genome](Lectures/D01T0930-20241028_I_GuidedTourOfHumGenome.pdf)<br/>[From DNA to biological function](Lectures/D01T0930-20241028_II_FromDNA2BiologicalFunction.pdf)<br/>[Genomics in biomedical research](Lectures/D01T0930-20241028_III_GenomicsInBiomedicalResearch.pdf)
| 12.00 - 13.15 | Lunch break
| 13.15 - 14.00 | [Sequencing and applications (TR)](Lectures/D01T1315-Sequencing_and_applications-TR-2024.pdf)
| 14.15 - 15.00 | [Illumina sequencing technology (AS)](Lectures/D01T1415-Illumina-AS.pdf)
| 15.15 - 16.00 | [PacBio sequencing technology (ATK)](Lectures/D01T1515-241028_IN_BIOSx000_ATK.pdf)
| **Tue 29 Oct**
| 09.15 - 10.00 | Oxford Nanopore sequencing technology (TH)
| 10.15 - 11.00 | [Library preparation (AS)](Lectures/D02T1015-Library-preparation-AS.pdf)
| 11.15 - 12.00 | [Experimental design (AS)](Lectures/D02T1115-Experimental-design-AS.pdf)
| 12.00 - 13.15 | Lunch break
| 13.15 - 14.00 | [Basic file formats (AS)](Lectures/D02T1315-Basic-file-formats-AS.pdf)
| 14.15 - 16.00 | [Access to computational resources (TR)](Lectures/D02T1415-Access_to_computational_resources-TR-2024.pdf)
| **Wed 30 Oct**
| 09.15 - 12.00 | [Data preprocessing (AS)](Lectures/D03T0915-Data-processing-AS.pdf)
| 12.00 - 13.15 | Lunch break
| 13.15 - 15.00 | Introduction to containers and analysis pipelines with Nextflow (AS)
| 15.15 - 16.00 | Introduction to Snakemake (EEB)
| **Thu 31 Oct**
| 09.15 - 12.00 | *De novo* genome assembly (KL)
| 12.00 - 13.15 | Lunch break
| 13.15 - 16.00 | *De novo* genome assembly, continued (KL)
| **Fri 1 Nov**
| 09.15 - 12.00 | *De novo* genome assembly, continued (KL)
| 12.00 - 13.15 | Lunch break
| 13.15 - 16.00 | *De novo* genome assembly, continued (KL)
| 16.00 - 17.00 | Pizza on campus
| &nbsp; | |
| **Week 45**
| **Mon 4 Nov**
| 09.15 - 12.00 | Introduction to variant calling (BS)
| 12.00 - 13.15 | Lunch break
| 13.15 - 15.00 | Introduction to variant calling, continued (BS)
| 15.15 - 17.00 | Human germline variant calling (YS)
| **Tue 5 Nov**
| 09.15 - 12.00 | Human germline variant calling, continued (YS)
| 12.00 - 13.15 | Lunch break
| 13.15 - 16.00 | Somatic variant calling and interpretation in the context of cancer (SN)
| **Wed 6 Nov**
| 09.15 - 12.00 | Microbiome analysis (NKN)
| 12.00 - 13.15 | Lunch break
| 13.15 - 17.00 | Multi-omic network analysis (NKN)
| **Thu 7 Nov**
| 09.15 - 12.00 | Transcriptomics (RS)
| 12.00 - 13.15 | Lunch break
| 13.15 - 17.00 | Small RNA transcriptomics (TBR)
| **Fri 8 Nov**
| 09.15 - 12.00 | Machine learning in computational biology (MP)
| 12.00 - 13.15 | Lunch break
| 13.15 - 16.00 | Machine learning in computational biology, continued (MP)
| 16.00 - 16.15 | Exam info and course evaluation (TR)
| &nbsp; | |
| **Week 50**
| **Mon 9 Dec**
| 09.15 - 15.00 | Oral presentation and exam for MSc students
| **Tue 10 Dec**
| 09.15 - 15.00 | Oral presentation and exam for PhD students
| 17.00 - 19.00 | Written exam for all


## Teachers

| Initials | Name                    | Affiliation                                                         |
| -------- | ----------------------- | ------------------------------------------------------------------- |
| AS       | Arvind Sundaram         | Norwegian Sequencing Centre, Oslo University Hospital
| ATK      | Ave Tooming-Klunderud   | CEES, Dept. of Biosciences, University of Oslo
| BS       | Bastiaan Star           | CEES, Dept. of Biosciences, University of Oslo
| EEB      | Einar E. Birkeland      | Dept. of Informatics / Pharmacy, University of Oslo
| KL       | Karin Lagesen           | Section for Epidemiology, Norwegian Veterinary Institute
| MP       | Milena Pavlovic         | Dept. of Informatics, University of Oslo
| NKN      | Nolan Keith Newman      | Centre for Molecular Medicine Norway (NCMM), University of Oslo
| RS       | Rolf Skotheim           | Dept. of Informatics, University of Oslo
| SN       | Sigve Nakken            | Dept. of Informatics, University of Oslo
| TH       | Thomas Haverkamp        | Section for Epidemiology, Norwegian Veterinary Institute
| TR       | Torbjørn Rognes         | Dept. of Informatics, University of Oslo
| TBR      | Trine B. Rounge         | Dept. of Pharmacy, University of Oslo
| YS       | Ying Sheng              | Dept. of Medical Genetics, Oslo University Hospital


## Contact information

* Torbjørn Rognes (Course coordinator), email torognes@ifi.uio.no


## Meals

Coffee and tea will be freely available each morning in the seminar room.

For lunch, we recommend the canteen in the other end of the building, [Ole-Johan eatery](https://www.sio.no/en/food-and-beverage/spisesteder/ole-johan-eatery), which serves a nice selection of warm and cold dishes.

There will be free pizza one afternoon.

## Requirements

* We recommend that participants should have a basic understanding of molecular biology as provided by an introductory course in bioinformatics, molecular biology, or genetics.

* No formal background in computer science is required, however, students must have a basic understanding of the unix shell. Students should take an introductory unix course beforehand if they do not have these skills.


## Computing facilities

All students must have a laptop with either Windows (version 10 or more recent), macOS, Linux or other Unix operating system. You need to be able to access the Eduroam wifi network. If you do not have a UiO or Eduroam account, a temporary Wifi account may be created for you. We advise you to bring an external mouse, and do not rely on touchpad/trackpad only.

The course will use computational resources on the Educloud Fox system at UiO. [Access to Educloud](Educloud/index.md) will be provided for all course participants. A [list of the software packages](software.md) that will be used is available.

If you are struggling with anything of the above, you must contact the course coordinator (see contact details on this page) as soon as possible, and at least one week before the start of the course.


## Curriculum and suggested reading

The curriculum consists of a set of scientific articles. The exam will contain questions on the material presented in these papers on the material covered during the course. Specific details of the curriculum, and a list of further suggested papers, can be found [here](Curriculum/index.md).


## Exam

The exam will be a combination of the following two parts:

 * an *individual* home project combined with an oral exam presentation of the home project followed by questioning / discussion on December 9th (MSc students) or 10th (PhD students). Students will present their work to 2-3 teachers, followed by some questioning (approx 20 minutes in total for each student). This home exam will be handed out to all participants after the lectures.
 * a 2 hour written digital exam at Silurveien 2 on December 10th for all students.

The exams are graded on a pass/fail scale.

Written exams from previous years:

* [2018](https://github.com/arvindsundaram/IN-BIOSx000/blob/2018/misc/written_exam.pdf)
* [2016](https://github.com/lexnederbragt/INF-BIOx121/blob/2017/misc/2016.pdf) (exam only)
* [2015](https://github.com/lexnederbragt/INF-BIOx121/tree/2015/exam) (with information on how the grading was done)
* [2014](https://wiki.uio.no/projects/clsi/images/f/f2/Examinfbio2014.pdf) (exam only)

There was no written exam in 2020 due to the pandemic.


## Lecture slides

[Lecture slides](Lectures/index.md)


## Teaching materials

List of further suggested papers, can be found [here](Curriculum/index.md).


## Previous editions of the course
Note: For previous editions of the IN-BIOS5000 / IN-BIOS9000 course please see the [2018](https://in-biosx000.readthedocs.io/en/2018/), [2020](https://in-biosx000.readthedocs.io/en/2020/), or [2022](https://uio-in-biosx000.readthedocs.io/en/2022/) websites. The course was not held in 2019 and 2021. For previous editions of the INF-BIO9121 and INF-BIO5121 course, please see the [2013](https://wiki.uio.no/projects/clsi/index.php/INF-BIOX120_H13), [2014](https://wiki.uio.no/projects/clsi/index.php/INF-BIOX121_H14), [2015](https://inf-biox121.readthedocs.io/en/2015/), [2016](https://inf-biox121.readthedocs.io/en/2016/) or [2017](https://inf-biox121.readthedocs.io/en/2017/) website. The 2012 version did not have a dedicated available website.
