# Curriculum and supporting material 2024

To be updated.


##### Molecular Biology

* Refer to Wikipedia for basic information.

* [NHGRI/NIH's Talking Glossary of Genomic and Genetic Terms](https://www.genome.gov/genetics-glossary)


##### High throughput sequencing (HTS)

* [The sequence of sequencers: The history of sequencing DNA](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/HTS_history.pdf), Heather JM _et. al._ (2016), Genomics.

* [High-Throughput Sequencing Technologies](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/HTS_technology.pdf), Reuter JA _et. al._ (2015) Molecular Cell.

* [Sequencing depth and coverage: key considerations in genomic analyses](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/HTS_Coverage_Depth.pdf), Sims D _et. al._ (2014) Nature Reviews.

* [Long-read human genome sequencing and its applications](https://doi.org/10.1038/s41576-020-0236-x) Logsdon, G.A., Vollger, M.R. & Eichler, E.E. (2020) Nat Rev Genet 21, 597–614.

* [Nanopore sequencing technology, bioinformatics and applications](https://doi.org/10.1038/s41587-021-01108-x), Wang, Y., Zhao, Y., Bollas, A. et al. (2021) Nat Biotechnol 39, 1348–1365.

* [The long reads ahead: _de novo_ genome assembly using the MinION](https://doi.org/10.12688/f1000research.12012.2), de Lannoy, C., de Ridder, D. & Risse, J. (2017) F1000Research 2017, 6:1083.


##### _De novo_ genome assembly

* [A field guide to whole-genome sequencing, assembly and annotation](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/DNA_1.pdf), Ekblom and Wolf (2014), Evolutionary Applications.

* [The Theory and Practice of Genome Sequence Assembly](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/DNA_2.pdf), Simpson and Pop (2015) Annual Review of Genomics and Human Genetics.

* [The present and future of de novo whole-genome assembly](https://doi.org/10.1093/bib/bbw096), Jang-il Sohn, Jin-Wu Nam (2018) Briefings in Bioinformatics, 19 (1), 23-40.

* [A survey on de novo assembly methods for single-molecular sequencing](https://dx.doi.org/10.1007/s40484-020-0214-5), Ying Chen, Chuan-Le Xiao (2020) Quant. Biol., 8, 3, 203-215.


##### Variant calling

* [Genotype and SNP calling from next-generation sequencing data](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/VC_1.pdf) Nielsen et al., Nature Reviews Genetics 2011. (This is a review paper that should be easily understandable once the student has taken the variant calling module.)

* [Exome sequencing identifies the cause of a mendelian disorder](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/VC_2.pdf) Ng et al., Nature Genetics 2010. (A paper reporting the first resolution of a mendelian disorder using exome capture.)

* [Variant calling: Considerations, practices, and developments](https://doi.org/10.1002/humu.24311), Zverinova, S., &  Guryev, V. (2022) Human Mutation,  43,  976-985.

* [Best practices for variant calling in clinical sequencing](https://doi.org/10.1186/s13073-020-00791-w), Koboldt, D.C. (2020) Genome Med 12, 91.

* [Advances in understanding cancer genomes through second-generation sequencing](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/SVC_1.pdf)
Getz, et al. (2010) Nature Reviews Genetics 2010.

* [A review of somatic single nucleotide variant calling algorithms for next-generation sequencing data](https://doi.org/10.1016/j.csbj.2018.01.003) Chang Xu (2018), Computational and Structural Biotechnology Journal, 16, 15-24.


##### Transcriptomics


##### Machine Learning

* [Deep learning in biomedicine](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/ML.pdf), Wainberg, et al. (2018) Nature Biotechnology.

* [DOME: recommendations for supervised machine learning validation in biology](https://doi.org/10.1038/s41592-021-01205-4), Walsh, I., Fishman, D., Garcia-Gasulla, D. et al. (2021) Nat Methods 18, 1122–1127.

* [Navigating the pitfalls of applying machine learning in genomics](https://doi.org/10.1038/s41576-021-00434-9), Whalen, S., Schreiber, J., Noble, W.S. et al. (2022) Nat Rev Genet 23, 169–181.
