# Curriculum and supporting material 2024

The curriculum of the course is the topics covered in the lectures and exercises.

The material below is supporting material aimed to help in the understanding of the topics.

The list will be updated.


##### Molecular Biology

* Refer to Wikipedia for basic information.

* [NHGRI/NIH's Talking Glossary of Genomic and Genetic Terms](https://www.genome.gov/genetics-glossary)


##### High throughput sequencing (HTS)

* [The sequence of sequencers: The history of sequencing DNA](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/HTS_history.pdf), Heather JM _et. al._ (2016), Genomics.

* [High-Throughput Sequencing Technologies](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/HTS_technology.pdf), Reuter JA _et. al._ (2015) Molecular Cell.

* [Sequencing depth and coverage: key considerations in genomic analyses](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/HTS_Coverage_Depth.pdf), Sims D _et. al._ (2014) Nature Reviews.

* [Long-read human genome sequencing and its applications](https://doi.org/10.1038/s41576-020-0236-x) Logsdon, G.A., Vollger, M.R. & Eichler, E.E. (2020) Nat Rev Genet 21, 597–614.

* [Nanopore sequencing technology, bioinformatics and applications](https://doi.org/10.1038/s41587-021-01108-x), Wang, Y., Zhao, Y., Bollas, A. et al. (2021) Nat Biotechnol 39, 1348–1365.

* [The long reads ahead: _de novo_ genome assembly using the MinION](https://doi.org/10.12688/f1000research.12012.2), de Lannoy, C., de Ridder, D. & Risse, J. (2017) F1000Research 2017, 6:1083.


##### Containers and workflow managers

* [Reproducible, scalable, and shareable analysis pipelines with bioinformatics workflow managers](https://doi.org/10.1038/s41592-021-01254-9) Wratten L, Wilm A & Göke J (2021) Nature Methods, 18, 1161-1168.


##### _De novo_ genome assembly

* [A field guide to whole-genome sequencing, assembly and annotation](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/DNA_1.pdf), Ekblom and Wolf (2014), Evolutionary Applications.

* [The Theory and Practice of Genome Sequence Assembly](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/DNA_2.pdf), Simpson and Pop (2015) Annual Review of Genomics and Human Genetics.

* [The present and future of de novo whole-genome assembly](https://doi.org/10.1093/bib/bbw096), Jang-il Sohn, Jin-Wu Nam (2018) Briefings in Bioinformatics, 19 (1), 23-40.

* [A survey on de novo assembly methods for single-molecular sequencing](https://dx.doi.org/10.1007/s40484-020-0214-5), Ying Chen, Chuan-Le Xiao (2020) Quant. Biol., 8, 3, 203-215.


##### Variant calling

* [Genotype and SNP calling from next-generation sequencing data](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/VC_1.pdf) Nielsen et al., Nature Reviews Genetics 2011. (This is a review paper that should be easily understandable once the student has taken the variant calling module.)

* [Exome sequencing identifies the cause of a mendelian disorder](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/VC_2.pdf) Ng et al., Nature Genetics 2010. (A paper reporting the first resolution of a mendelian disorder using exome capture.)

* [Variant calling: Considerations, practices, and developments](https://doi.org/10.1002/humu.24311), Zverinova, S., &  Guryev, V. (2022) Human Mutation,  43,  976-985.

* [Best practices for variant calling in clinical sequencing](https://doi.org/10.1186/s13073-020-00791-w), Koboldt, D.C. (2020) Genome Med 12, 91.

* [Advances in understanding cancer genomes through second-generation sequencing](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/SVC_1.pdf)
Getz, et al. (2010) Nature Reviews Genetics 2010.

* [A review of somatic single nucleotide variant calling algorithms for next-generation sequencing data](https://doi.org/10.1016/j.csbj.2018.01.003) Chang Xu (2018), Computational and Structural Biotechnology Journal, 16, 15-24.


##### Microbiome analysis

* [Normalization and microbial differential abundance strategies depend upon data characteristics](https://doi.org/10.1186/s40168-017-0237-y) Weiss et al (2017) Microbiome 5, 27 (2017).

* [QIIME2 Moving pictures tutorial](https://docs.qiime2.org/2024.10/tutorials/moving-pictures/)


##### Network analysis

* [Reverse enGENEering of Regulatory Networks from Big Data: A Roadmap for Biologists](https://doi.org/10.4137/BBI.S12467) Dong et al. (2015) Bioinformatics and Biology Insights, 2015, 9.

* [Transkingdom Network Analysis (TkNA): a systems framework for inferring causal factors underlying host–microbiota and other multi-omic interactions](https://doi.org/10.1038/s41596-024-00960-w) Newman et al. (2024) Nat Protoc 19, 1750–1778.

* [Passing messages between biological networks to refine predicted interactions](https://doi.org/10.1371/journal.pone.0064832) Glass et al. (2013) PLoS One, 8, e64832.

* [The Network Zoo: a multilingual package for the inference and analysis of gene regulatory networks](https://doi.org/10.1186/s13059-023-02877-1) Ben Guebila et al. (2023) Genome Biol, 24, 45.


##### Transcriptomics

[A survey of best practices for RNA-seq data analysis](https://doi.org/10.1186/s13059-016-0881-8) Conesa et al. (2016) Genome Biology, 17, 13.

[The hitchhikers’ guide to RNA sequencing and functional analysis](https://doi.org/10.1093/bib/bbac529) Chen et al. (2023) Briefings in Bioinformatics, 24, bbac529.


##### Small RNA transcriptomics

[A comprehensive profile of circulating RNAs in human serum](https://doi.org/10.1080/15476286.2017.1403003) Umu et al. (2017) RNA Biology, 15(2), 242–250.

[miEAA 2023: updates, new functional microRNA sets and improved enrichment visualizations](https://doi.org/10.1093/nar/gkad392) Aparicio-Puerta et al. (2023) Nucleic Acids Research, 51, W1, W319–W325.


##### Machine Learning

* [Deep learning in biomedicine](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/ML.pdf), Wainberg, et al. (2018) Nature Biotechnology.

* [DOME: recommendations for supervised machine learning validation in biology](https://doi.org/10.1038/s41592-021-01205-4), Walsh, I., Fishman, D., Garcia-Gasulla, D. et al. (2021) Nat Methods 18, 1122–1127.

* [Navigating the pitfalls of applying machine learning in genomics](https://doi.org/10.1038/s41576-021-00434-9), Whalen, S., Schreiber, J., Noble, W.S. et al. (2022) Nat Rev Genet 23, 169–181.
